nightingale_interface/screens/home_screen.py

```diff
@@ -8,50 +8,70 @@
 from kivy.uix.screenmanager import SlideTransition, NoTransition
 from screens.screen_config import ScreenConfig as cfg
 
+# TEMPORARY UNTIL INCLUDES ARE FIXED
+import sys
 
-class HomeScreen:
-    home_name = "homescreen"
+sys.path.append("..")
+# TEMPORARY UNTIL INCLUDES ARE FIXED
+
+from nightingale_ros_bridge.src.nightingale_ros_bridge.bridge_interface_config import (
+    BridgeConfig,
+    UserInputs,
+)
+

```

Comment:
This is a good temporary fix, eventaully we can package the app to run as an executable though and the imports will be handeled in the build process. https://kivy.org/doc/stable/guide/packaging-windows.html